caravel/utils.py

```diff
@@ -414,3 +419,31 @@ def generic_find_constraint_name(table, columns, referenced, db):
                 fk.referred_table.name == referenced and
                 set(fk.column_keys) == columns):
             return fk.name
+
+
+class timeout(object):
```

Comment:
timeout will be used as a context manager (`with` clause), those are usually lower case